docs/wiki/[Variables]-policy_compliance_message_not_supported_definitions.md

```diff
@@ -0,0 +1,34 @@
+<!-- markdownlint-disable first-line-h1 -->
+## Overview
+
+[**policy_compliance_message_not_supported_definitions**](#overview) `bool` (optional)
+
+If specified, provides a list of built in policy definitions that do not support compliance messages. Only policies with the mode of 'Indexed' or 'All' support compliance messages. Any assignments with these policies will not have the default compliance message applied.
+
+## Default value
+
+```
```

Comment:
Please check this entry as the linter is complaining about it:

```shell
/github/workspace/docs/wiki/User-Guide.md:102:1 MD053/link-image-reference-definitions Link and image reference definitions should be needed [Unused link or image reference definition: "wiki_policy_compliance_message_not_supported_definitions"] [Context: "[wiki_policy_compliance_messag..."]
```

Reply:
> 

@krowlandson The `policy_compliance_message_default_enabled` variable is designed to support that use case.